roles/openshift_examples/templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+EXAMPLES_BASE=$(pwd)/files/examples
+find files/examples -name '*.json' -delete
+TEMP=`mktemp -d`
+pushd $TEMP
+wget https://github.com/openshift/origin/archive/master.zip -O origin-master.zip
+wget https://github.com/openshift/django-ex/archive/master.zip -O django-ex-master.zip
+wget https://github.com/openshift/rails-ex/archive/master.zip -O rails-ex-master.zip
+wget https://github.com/openshift/nodejs-ex/archive/master.zip -O nodejs-ex-master.zip
+wget https://github.com/openshift/dancer-ex/archive/master.zip -O dancer-ex-master.zip
+wget https://github.com/openshift/cakephp-ex/archive/master.zip -O cakephp-ex-master.zip
+wget https://github.com/jboss-openshift/application-templates/archive/master.zip -O application-templates-master.zip
+unzip origin-master.zip
+unzip django-ex-master.zip
+unzip rails-ex-master.zip
+unzip nodejs-ex-master.zip
+unzip dancer-ex-master.zip
+unzip cakephp-ex-master.zip
+unzip application-templates-master.zip
+cp origin-master/examples/db-templates/* ${EXAMPLES_BASE}/db-templates/
+cp origin-master/examples/image-streams/* ${EXAMPLES_BASE}/image-streams/
+cp django-ex-master/openshift/templates/* ${EXAMPLES_BASE}/quickstart-templates/
```

Comment:
This script is meant to help maintain the content in the ansible repo, its not used directly by ansible.

Sent from my Android phone using Symantec TouchDown (www.symantec.com)


-----Original Message-----
From: Rodolfo Carvalho [notifications@github.com]
Received: Tuesday, 23 Jun 2015, 6:40AM
To: openshift/openshift-ansible [openshift-ansible@noreply.github.com]
CC: Scott Dodson [sdodson@redhat.com]
Subject: Re: [openshift-ansible] Openshift examples (#289)

> +wget https://github.com/openshift/django-ex/archive/master.zip -O django-ex-master.zip
> +wget https://github.com/openshift/rails-ex/archive/master.zip -O rails-ex-master.zip
> +wget https://github.com/openshift/nodejs-ex/archive/master.zip -O nodejs-ex-master.zip
> +wget https://github.com/openshift/dancer-ex/archive/master.zip -O dancer-ex-master.zip
> +wget https://github.com/openshift/cakephp-ex/archive/master.zip -O cakephp-ex-master.zip
> +wget https://github.com/jboss-openshift/application-templates/archive/master.zip -O application-templates-master.zip
> +unzip origin-master.zip
> +unzip django-ex-master.zip
> +unzip rails-ex-master.zip
> +unzip nodejs-ex-master.zip
> +unzip dancer-ex-master.zip
> +unzip cakephp-ex-master.zip
> +unzip application-templates-master.zip
> +cp origin-master/examples/db-templates/* ${EXAMPLES_BASE}/db-templates/
> +cp origin-master/examples/image-streams/* ${EXAMPLES_BASE}/image-streams/
> +cp django-ex-master/openshift/templates/* ${EXAMPLES_BASE}/quickstart-templates/

Since we copy the templates from master of each repository, why are we including a [hard copy](https://github.com/openshift/openshift-ansible/pull/289/files#diff-3ce22be1a86be6c1a8e71099f1d31fed) of those as well?

---
Reply to this email directly or view it on GitHub:
https://github.com/openshift/openshift-ansible/pull/289/files#r33028621